server/run.sh

```diff
@@ -13,6 +13,7 @@ do
     "--test" )
       python -m pytest;;
     "--dev" )
+      redis-server
```

Comment:
Didn't know that this could be done. It seems to work fine, running the redis-server and the python server at the same time, but it looks to block the output of the python server. Is there a way to fix this? 

Also if this is being done, the documentation for running the python server locally will need to be changed.

Reply:
Good catch I didn't realize this was changed when I removed the production environment. I think it makes more sense to remove this.